### ISSUE TITLE
Not syncing with Emby 4.8.0.19

### ISSUE THREAD

**Issue by carbonbaseunit** [AAndyProgram/PlexEmbySync#2]
Emby has stopped syncing with PlexEmbySync.  You can authenticate the user and program, but nothing happens when you play a program from Emby.  From Plex it works as it should.

**Comment by AAndyProgram** [comment]
A program? Do you mean an episode of a TV-Show? Some episodes don't sync. I think it's an encoding error. Try playing another episode of a TV-Show that you're sure has been synced.

**Comment by AAndyProgram** [comment]
Btw, 4.8.0.19 is a beta version. I think beta versions may contain bugs. I'm using version [4.7.9.0](https://github.com/MediaBrowser/Emby.Releases/releases/tag/4.7.9.0).
Of course, maybe something has changed in this version. But when it becomes stable, I'll use it, and if PlexEmbySync doesn't work, I'll fix it.

**Comment by carbonbaseunit** [comment]
I'm sorry, man.  I didn't know you had replied.  I flubbed my terms.  Your program connects to Emby, but nothing at all shows up in your UI when I play any movie or TV show that I have synced on Emby nor does Plex show any changes corresponding to changes with Emby .  When I play a movie or TV show with Plex it works as it should.  I have been using Emby Beta for a long time without any issues.  Maybe there was a change in the code recently that broke compatibility?  Nothing is showing up in the log.
Thank you for responding and, again, sorry I didn't respond in a timely fashion.
